src/core/thread.cc

```diff
@@ -48,28 +48,41 @@ namespace seastar {
 thread_local jmp_buf_link g_unthreaded_context;
 thread_local jmp_buf_link* g_current_context;
 
-#ifdef SEASTAR_ASAN_ENABLED
+#if defined(SEASTAR_ASAN_ENABLED) || defined(SEASTAR_TSAN_ENABLED)
 
 namespace {
 
-#ifdef SEASTAR_HAVE_ASAN_FIBER_SUPPORT
+#if defined(SEASTAR_ASAN_ENABLED) && defined(SEASTAR_HAVE_ASAN_FIBER_SUPPORT)
```

Comment:
because `#if` at higher level has changed
```diff
- #ifdef SEASTAR_ASAN_ENABLED
+ #if defined(SEASTAR_ASAN_ENABLED) || defined(SEASTAR_TSAN_ENABLED)
```

Reply:
I see the comment below about it